packages/renderer/src/lib/objects/KubernetesColumnName.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+import type { KubernetesNamespacedObjectUI, KubernetesObjectUI } from './KubernetesObjectUI';
+
+interface Props {
+  object: KubernetesObjectUI;
+}
+
+let { object }: Props = $props();
+
+function isNamespaced(object: KubernetesObjectUI): object is KubernetesNamespacedObjectUI {
+  return 'namespace' in object;
+}
+
+async function openDetails(): Promise<void> {
+  if (isNamespaced(object)) {
+    await window.navigateToRoute('kubernetes', [{ kind: object.kind, name: object.name, namespace: object.namespace }]);
+  } else {
+    await window.navigateToRoute('kubernetes', [{ kind: object.kind, name: object.name }]);
```

Comment:
I did not notice before, but why do we need to pass an array? it only accept a single argument?

Reply:
Good catch (and thanks for helping me figure it out). navigateToRoute was missing a spread, fixed in new commit.